spec/services/twilio_service_spec.rb

```diff
@@ -50,28 +50,28 @@
       TwilioService.new
     end
 
-    it 'sends an OTP from the test number when pt_mode is true' do
+    it 'sends an OTP from the test number when pt_mode is true', sms: true do
       expect(FeatureManagement).to receive(:pt_mode?).at_least(:once).and_return(true)
       SmsSenderOtpJob.perform_now(user)
 
       expect(messages.first.from).to eq '+15005550006'
     end
 
-    it 'sends an OTP from the real number when pt_mode is false' do
+    it 'sends an OTP from the real number when pt_mode is false', sms: true do
       expect(FeatureManagement).to receive(:pt_mode?).at_least(:once).and_return(false)
       SmsSenderOtpJob.perform_now(user)
 
       expect(messages.first.from).to eq '+19999999999'
```

Comment:
pull from ~~application.yml~~ secrets.yml and remove unused keys from application.yml.example